docs/release/release_0_6_0.md

```diff
@@ -203,44 +363,89 @@ issues](https://github.com/napari/napari) or on our [Zulip chat room](https://na
+- Update `coverage`, `fsspec`, `hypothesis`, `pydantic`, `rich`, `superqt`, `tensorstore`, `tifffile`, `xarray` ([#7767](https://github.com/napari/napari/pull/7767))
+- improve reporting errors in shape triangulation ([#7768](https://github.com/napari/napari/pull/7768))
+- ci(dependabot): bump docker/login-action from 3.3.0 to 3.4.0 in the actions group ([#7769](https://github.com/napari/napari/pull/7769))
+- Remove deprecated QtViewer.view and QtViewer.camera properties ([#7783](https://github.com/napari/napari/pull/7783))
+- Update task.md ([#7789](https://github.com/napari/napari/pull/7789))
+- Postpone QtViewer deprecation to 0.7.0 ([#7791](https://github.com/napari/napari/pull/7791))
+- Update `fsspec`, `hypothesis`, `lxml`, `pydantic`, `virtualenv` ([#7792](https://github.com/napari/napari/pull/7792))
+- Remove deprecated interpolation argument from add_image ([#7793](https://github.com/napari/napari/pull/7793))
+- Remove deprecated type kwarg from Events ([#7794](https://github.com/napari/napari/pull/7794))
+- [pre-commit.ci] pre-commit autoupdate ([#7798](https://github.com/napari/napari/pull/7798))
+- Update camera orientation tooltips for GUI preferences and camera widget ([#7813](https://github.com/napari/napari/pull/7813))
 - Add codespell support (config, workflow to detect/not fix) and make it fix few typos ([docs#587](https://github.com/napari/docs/pull/587))
 - Update CircleCI config.yml to use napari docs and gallery dependency groups ([docs#590](https://github.com/napari/docs/pull/590))
 - Update build_and_deploy.yml to use napari docs dependency group ([docs#591](https://github.com/napari/docs/pull/591))
 - Replace two manual screenshots of the viewer (launch_cli_empty and launch_cli_image) with nbscreenshots ([docs#606](https://github.com/napari/docs/pull/606))
 - Update conf.py to address pygment -> pygments deprecation ([docs#607](https://github.com/napari/docs/pull/607))
 - Update CircleCI config to python orb and image ([docs#611](https://github.com/napari/docs/pull/611))
+- Use env var to control if notebooks are executed and add docs, slim, slimfast, and slimgallery variants to Makefile ([docs#646](https://github.com/napari/docs/pull/646))
+- [bugfix] Fix linkcheck.yml for change to dependencies and removal of req.txt ([docs#654](https://github.com/napari/docs/pull/654))
+- Squash deployed gh-pages, relying on this repo for history ([docs#658](https://github.com/napari/docs/pull/658))
+- Revert "Squash deployed gh-pages, relying on this repo for history (#658)" ([docs#663](https://github.com/napari/docs/pull/663))
 
 
-## 14 authors added to this release (alphabetical)
+## 19 authors added to this release (alphabetical)
 
 (+) denotes first-time contributors 🥳
 
-- [Andrew Sweet](https://github.com/napari/napari/commits?author=andy-sweet) - @andy-sweet
+- [andrew sweet](https://github.com/napari/napari/commits?author=andy-sweet) - @andy-sweet
```

Comment:
Why did Andy and Clement lose their capital letters :sweat_smile:

Reply:
🤔 something weird about their GH profiles. As far as I remember, this is obtained from the GH API... But their profile pages are correct... 🤔 Anyway, will manually fix for now.